### PR TITLE
Show last score and high score

### PR DESCRIPTION
**Pull request by tjorim**
## Description

This will show and save the highscore, as well as the score from last game.

There were also multipe generalStyle variables which have been put together
and not-closed `<span>`-tags are fixed now. Removes previous zoom-function.
## Motivation and Context

This will partially fix issue #38 (more some sort of feature)
and help with the implementation of #129 .

The code could be adjusted to save multiple score
in an array with day and maybe bot version.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ermiyaeskandary** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/189/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

File: bot.user.js [hunk 1]
```diff
@@ -1138,34 +1152,44 @@ window.sosBackup = sos;
...
         '(X) Change skin </span>';
-    window.quickResp_overlay.innerHTML = generalStyle +
+    window.quickResp_overlay.innerHTML = window.spanstyle +
         '(ESC) Quick Respawn </span>';
     // eslint-disable-next-line no-undef
-    window.version_overlay.innerHTML = generalStyle + 'Version: ' + GM_info
-        .script.version;
-
+    window.version_overlay.innerHTML = window.spanstyle +
+		'Version: ' + GM_info.script.version + '</span>';
```
Comment (ermiyaeskandary) [inline bot.user.js]: Add this above
`// eslint-disable-next-line no-undef`

**Comment by Drflash55** [comment]
Approved. This seems like a useful feature...

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/189/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Comment by ermiyaeskandary** [comment]
@tjorim Can you fix indentation ?

**Comment by Drflash55** [comment]
Alrighty, we're good to go... ~~as soon as github is done checking the automatic merge status...~~

You can merge when ready...

File: bot.user.js [hunk 2]
```diff
@@ -386,17 +378,14 @@ var bot = (function() {
...
                 bot.connectBot();
-                if (document.querySelector('div#lastscore').childNodes
-                    .length > 1) {
-                    window.scores.push(parseInt(document.querySelector(
-                        'div#lastscore').childNodes[1].innerHTML));
-                }
+                /* The lastscore should be checked here in the future,
+                but not only when bot, just when not playing.
+                THIS SHOULD ONLY BE EXECUTED ONCE! */
+                userInterface.saveScore(); // Checks highscore
```
Comment (ermiyaeskandary) [inline bot.user.js:384]: Hm ?

**Pull request merged**